src/containers/Network/NetworkTabs.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { useRouteMatch } from 'react-router'
+import Tabs from '../shared/components/Tabs'
+
+interface Props {
+  selected: string
+}
+
+const NetworkTabs = (props: Props) => {
+  const { selected } = props
+  const { path = '/' } = useRouteMatch()
+  const tabs = ['nodes', 'validators', 'upgrade-status']
```

Comment:
This is defined manually in multiple spots - is there a single place we could define these tabs? (That way the code can follow the [DRY principle](https://www.digitalocean.com/community/tutorials/what-is-dry-development), and it's easier to update later on :) )

Reply:
In this case it would have been 4 and is now 2.  The way this page works is very weird usually the content above the tabs does not change when you change tabs.

Reply:
What would have been 4? Not quite sure what the numbers are in response to

Reply:
It was in 3 places before this PR.  A new tab was added which would have been a new copy paste but this component was create to reuse code for 3 of the instances.

Reply:
I fixed the Nodes.jsx file so these definitions are now only in one place.